src/Input.js

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components'
+import { space, theme, propTypes } from 'styled-system'
+import PropTypes from 'prop-types'
+
+const borders = ({ color, theme }) => {
+  const borderColor = (color) ? theme.colors[color] : theme.colors.borderGray
```

Comment:
I don't think we should import theme here but rather use the theme as prop coming from the `ThemeProvider`

Reply:
borders is not a styled component, it's just a function and doesn't have access to the props unless I explicitly pass it in on line 44. @jxnblk, what should I do?

Reply:
Yeah, you shouldn't need to import `theme` ever. The way you pass the function to styled-components on line 44 means that styled-components will pass the full `props` object in here, adding in `props.theme` – you'll still need to use this within a ThemeProvider for it to work, but for right now, that's the pattern we're following

Reply:
Oh right, forgot, that "theme" I'm importing and using isn't the one from the design-system, it's the helper function from styled system @hakimelek